subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -117,6 +117,55 @@ PMD has been updated to https://pmd.github.io/pmd-6.48.0/pmd_release_notes.html[
 When configuring an executable explicitly for link:{groovyDslPath}/org.gradle.api.tasks.compile.ForkOptions.html#org.gradle.api.tasks.compile.ForkOptions:executable[`JavaCompile`] or link:{groovyDslPath}/org.gradle.api.tasks.testing.Test.html#org.gradle.api.tasks.testing.Test:executable[`Test`] tasks, Gradle will now emit an error if this executable does not exist.
 In the past, the task would be executed with the default toolchain or JVM running the build.
 
+==== Strongly typed dependency declarations for Test Suites
+
+The `dependencies` block used to configure Test Suites and located at `testing.suites.<MY_SUITE_NAME>.dependencies` continues to diverge from the top-level `dependencies` block.
+This is part of an ongoing effort to improve this `@Incubating` API to make it more discoverable and strongly typed.
+This change should have the benefit of making the `testing` DSL easier to work with in your IDE.
+
+On consequence of this is that buildsscripts which previously added project dependencies to a suite using:
```

Comment:
```suggestion
==== Added strongly typed dependency declarations to test suites

As part of the ongoing effort to improve the incubating Test Suites `dependencies` block, dependency declarations are now strongly typed.
In some case, this requires syntax changes.
For example, build scripts that previously added test suite dependencies with the following syntax:
```
Suggestion: the first three sentences of this description feel a bit sales-y to me -- I think we can streamline them into a single sentence. Additionally, we currently bury the lede of syntax breaking changes -- better to mention that as early as possible. And by introducing that concept explicitly sooner, we can introduce the example as an example to help users understand *why* they're reading this code block -- right now, it's unclear what the code block is trying to accomplish unless you read the entire section, then come back to it.